src/components/Quote/Quote.tsx

```diff
@@ -0,0 +1,20 @@
+"use client";
+import styles from "./Quote.module.scss";
+
+type Props = {
+  quote: string;
+  name: string;
+  type: string;
+};
+
+export function Quote(props: Props) {
+  return (
```

Comment:
you can destructure props above line 11

`const { quote, name, type } = props;`

Reply:
Didnt know you could destructure in TS... Thanks!